src/shapes/IText/DraggableTextDelegate.ts

```diff
@@ -169,30 +182,14 @@ export class DraggableTextDelegate {
   onDragStart(e: DragEvent): boolean {
     this.__dragStartFired = true;
     const target = this.target;
+    const { selectionStart, selectionEnd } = target;
     const active = this.isActive();
     if (active && e.dataTransfer) {
-      const selection = (this.__dragStartSelection = {
-        selectionStart: target.selectionStart,
-        selectionEnd: target.selectionEnd,
-      });
-      const value = target._text
-        .slice(selection.selectionStart, selection.selectionEnd)
-        .join('');
-      const data = { text: target.text, value, ...selection };
-      e.dataTransfer.setData('text/plain', value);
-      e.dataTransfer.setData(
```

Comment:
if you resort to having 
```
e.dataTransfer.setData(this.getFormattedClipboardData(value, styles))
```

you can just add a standard function that by default return `{ value, styles }` and the optinal one that returns what it has to return.

Who wants to use it will add it to the prototype in their own app repacing the old one, or will override the class if they prefer.